server/middleware/recovery_test.go

```diff
@@ -13,12 +13,10 @@ import (
 
 func TestRecovery(t *testing.T) {
 	var (
-		called  bool
 		errBoom = errors.New("boom")
 	)
 
 	ctx := reporter.WithReporter(context.Background(), reporter.ReporterFunc(func(ctx context.Context, err error) error {
-		called = true
```

Comment:
❔  Instead of deleting this, should we assert that `called` is true after the unit under test has executed?

Reply:
I'm not sure what `called` represents here, though (possibly an aspect of the context that's expected to be preserved and passed through?). The rest of this test seems to be checking that the error is handled gracefully and reported. After clicking through remind101's pile of [bespoke wrappers around go standard lib stuff](https://github.com/remind101/pkg) and not finding any more clarity there, I'm going to leave this as-is (unused, and now deleted).